runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -260,13 +261,38 @@ impl<T: Config> Pallet<T> {
 			parent_header,
 			mut disputes,
 		} = data;
+		sp_io::init_tracing();
+		let mut weight_metric = CounterVec::new(
+			"parachain_inherent_data_weight",
+			"Inherent data weight before and after filtering",
+			&["when"],
+		);
 
-		let parent_header_hash = parent_header.hash();
+		let mut bitfields_processed_metric = Counter::new(
+			"parachain_inherent_data_bitfields_processed",
+			"Counts the number of bitfields processed in `enter_inner`.",
+		);
+
+		let mut candidates_processed_metric = CounterVec::new(
+			"parachain_inherent_data_candidates_processed",
+			"Counts the number of parachain block candidates processed in `enter_inner`.",
+			&["category"],
+		);
+
+		let mut dispute_sets_processed_metric = CounterVec::new(
+			"parachain_inherent_data_dispute_sets_processed",
+			"Counts the number of dispute statements sets processed in `enter_inner`.",
+			&["category"],
+		);
+
+		let mut disputes_included_metric = Counter::new(
+			"parachain_inherent_data_disputes_included",
+			"Counts the number of dispute statements sets included in a block in `enter_inner`.",
+		);
 
 		log::debug!(
 			target: LOG_TARGET,
-			"[enter_inner] parent_header={:?} bitfields.len(): {}, backed_candidates.len(): {}, disputes.len(): {}",
-			parent_header_hash,
+			"[enter] bitfields.len(): {}, backed_candidates.len(): {}, disputes.len() {}",
```

Comment:
Why is this removed? That was explicitly added since that helps to lookup the block in the UI

Reply:
I don't know, seems to be an error on my side.

Reply:
I guess this is merge damage. I'll fix it.